index.html

```diff
@@ -86,7 +86,7 @@ <h2 class="portfolio_header">My Recent Work</h2>
           </div>
 
           <section class="portfolio_work">
-            <div class="work_item">
+            <!-- <div class="work_item">
```

Comment:
- [OPTIONAL] Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻